apps/andi/assets/css/edit.scss

```diff
@@ -138,6 +138,23 @@ $component-header-height: 7.5rem;
     margin-left: 1rem;
 }
 
+.unsaved-changes-modal--visible {
```

Comment:
@cameronmarsh do you think it's worthwhile to do a sass mixin for the modal to dry this up?

Reply:
yeah I think that it would be worth it. Shouldn't be too hard

Reply:
actually there's already a class so a mixin might not even be necessary